src/pages/PageProfile.vue

```diff
@@ -2,20 +2,19 @@
   <ApolloQuery
     :query="displayProfile"
     :variables="variables"
+    :fetchPolicy="viewAs.filter ? 'no-cache' : 'cache-first'"
     clientId="mutationClient"
   >
     <template slot-scope="{ result: { loading, data, error } }">
+      <PreviewAs v-if="showPreviewAs" :viewAsFilter="viewAs"></PreviewAs>
```

Comment:
I think it might be helpful to put this preview as tag inside the data template. When I pulled loaded this page up, the preview as button came on the page faster than the profile loaded. Seems like you shouldn't be able to interact with previewAs if the profile isn't loaded yet.

https://drive.google.com/open?id=1Piwjn7y97snbMop-QoZOqFRCLOibVjOz

Reply:
:+1: